sdcm/sct_runner.py

```diff
@@ -62,185 +87,285 @@ def instance_root_disk_size(test_duration) -> int:
+            LOGGER.info("Copying %s to %s...\nNOTE: it can take 5-15 minutes.",
+                        self.image_name, self.region_name)
+            self._copy_source_image_to_region()
+            LOGGER.info("Done.")
+        else:
+            LOGGER.info("No need to copy SCT Runner image since it already exists in `%s'", self.region_name)
+
+    @abstractmethod
+    def _get_base_image(self, image: Optional[Any] = None) -> Any:
+        ...
+
+    def create_instance(self, test_id: str, test_duration: int, instance_type: str = "") -> Any:
         LOGGER.info("Creating SCT Runner instance...")
         image = self.image
         if not image:
             LOGGER.error("SCT Runner image was not found in %s! "
-                         "Use hydra create-runner-image --cloud-privider %s --region %s",
+                         "Use `hydra create-runner-image --cloud-provider %s --region %s'",
                          self.region_name, self.cloud_provider, self.region_name)
-            sys.exit(1)
-        lt_datetime = datetime.datetime.now(tz=pytz.utc)
+            return None
         return self._create_instance(
             instance_type=instance_type or self.instance_type(test_duration=test_duration),
             base_image=self._get_base_image(self.image),
-            tags_list=[
-                {"Key": "Name", "Value": self.RUNNER_NAME},
-                {"Key": "TestId", "Value": test_id},
-                {"Key": "NodeType", "Value": self.NODE_TYPE},
-                {"Key": "RunByUser", "Value": get_username()},
-                {"Key": "keep", "Value": str(ceil(test_duration / 60) + 6)},
-                {"Key": "keep_action", "Value": "terminate"},
-                {"Key": "launch_time", "Value": lt_datetime.strftime("%B %d, %Y, %H:%M:%S")},
-            ],
+            tags={
+                "TestId": test_id,
+                "NodeType": self.NODE_TYPE,
+                "RunByUser": get_username(),
+                "keep": str(ceil(test_duration / 60) + 6),  # keep SCT Runner for 6h more than test_duration
+                "keep_action": "terminate",
+            },
             instance_name=f"{self.image_name}-instance-{test_id[:8]}",
-            region_az=region_az,
+            region_az=self.region_az(
+                region_name=self.region_name,
+                availability_zone=self.availability_zone,
+            ),
             test_duration=test_duration,
         )
 
 
 class AwsSctRunner(SctRunner):
-    """Provisions and configures the SCT runner on AWS"""
+    """Provision and configure the SCT Runner on AWS."""
     BASE_IMAGE = "ami-0ffac660dd0cb2973"  # ubuntu/images/hvm-ssd/ubuntu-focal-20.04-amd64-server-20200609
     SOURCE_IMAGE_REGION = "eu-west-2"  # where the source Runner image will be created and copied to other regions
+    IMAGE_BUILDER_INSTANCE_TYPE = "t2.small"
+    REGULAR_TEST_INSTANCE_TYPE = "t3.large"  # has 7h 12m CPU burst
```

Comment:
Is this really correct info -> "7h 12m CPU" ?
According to the AWS docs it has 2 CPU cores with 36m credits each. 
So, it must be 1 (one) hour and 12 minutes.